.travis.yml

```diff
@@ -70,67 +77,326 @@ jobs:
       after_success: false
       after_failure: false
       if: branch = master AND type = push
+
+    - stage: test-languages
```

Comment:
this job is holding up the build, as it takes longer than the others in test-languages; maybe move to `test-other-versions`

the others finish around 20mins - the windows job takes 27 mins

Reply:
It has been moved into `Test-other-versions` and the other jobs in this group take appox 27 mins, however that is largely because the R jobs are very inefficient, so that may need to be revisited if the R jobs are improved.